evennia/utils/opclasses.py

```diff
@@ -0,0 +1,278 @@
+import datetime as _dt
```

Comment:
For clarity, naming this module  `optionclasses.py` to be in line with the naming scheme of e.g. `inlinefuncs.py` etc which don't abbreviate the first word.